dump/mapping/prepbufr_adpupa.yaml

```diff
@@ -0,0 +1,315 @@
+    # MetaData
+    timestamp:
+      timeoffset:
+        timeOffset: "*/PRSLEVEL/DRFTINFO/HRDR"
+        transforms:
+          - scale: 3600
+        referenceTime: "1970-01-01T00:00:00Z"
+    prepbufrDataLevelCategory:
+      query: "*/PRSLEVEL/CAT"
+    obsTimeMinusCycleTime:
+      query: "*/PRSLEVEL/DRFTINFO/HRDR"
+    longitude:
+      query: "*/PRSLEVEL/DRFTINFO/XDR"
+      transforms:
+        - wrap: [ -180.0, 180.0 ]
+    latitude:
+      query: "*/PRSLEVEL/DRFTINFO/YDR"
+    stationIdentification:
+      query: "*/SID"
+    stationElevation:
+      query: "*/ELV"
+    pressure:
+      query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/POB"
+      type: float
+      transforms:
+        - scale: 100
+    height:
+      query: "*/PRSLEVEL/Z___INFO/Z__EVENT{1}/ZOB"
+      type: float
+    temperatureEventCode:
+      query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TPC"
+        
+    # ObsValue
+    airTemperature:
+      query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TOB"
+      transforms:
+        - offset: 273.15
+    dewPointTemperature:
+      query: "*/PRSLEVEL/Q___INFO/TDO"
+      transforms:
+        - offset: 273.15
+    virtualTemperature:
+      query: "*/PRSLEVEL/T___INFO/TVO"
+      transforms:
+        - offset: 273.15
+    windEastward:
+      query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/UOB"
+    windNorthward:
+      query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/VOB"
+    specificHumidity:
+      query: "*/PRSLEVEL/Q___INFO/Q__EVENT{1}/QOB"
+      type: float
+      transforms:
+        - scale: 0.000001
+
+    # QualityMark
+    pressureQualityMarker:
+      query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/PQM"
+    heightQualityMarker:
+      query: "*/PRSLEVEL/Z___INFO/Z__EVENT{1}/ZQM"
+    airTemperatureQualityMarker:
+      query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TQM"
+    virtualTemperatureQualityMarker:
+      query: "*/PRSLEVEL/T___INFO/T__EVENT{1}/TQM"
+    windQualityMarker:
+      query: "*/PRSLEVEL/W___INFO/W__EVENT{1}/WQM"
+    specificHumidityQualityMarker:
+      query: "*/PRSLEVEL/Q___INFO/Q__EVENT{1}/QQM"
+
+    # ObsError
+    pressureError:
+      query: "*/PRSLEVEL/P___INFO/P__BACKG/POE"
+      transforms:
+        - scale: 100
+    airTemperatureError:
+      query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+      transforms:
+        - offset: 273.15
+    virtualTemperatureError:
+      query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+      transforms:
+        - offset: 273.15
+    windError:
+      query: "*/PRSLEVEL/W___INFO/W__BACKG/WOE"
+    specificHumidityError:
+      query: "*/PRSLEVEL/Q___INFO/Q__BACKG/QOE"
+
+
+encoder:
+  globals:
+    - name: "dataOriginalFormatSpec"
+      type: string
+      value: "prepbufr"
+
+    - name: "platforms"
+      type: string
+      value: "ADPUPA"
+
+    - name: "source"
+      type: string
+      value: "prepBUFR"
+
+    - name: "description"
+      type: string
+      value: "UPPER-AIR (RAOB, PIBAL, RECCO, DROPS) REPORTS"
+
+    - name: "dataProviderOrigin"
+      type: string
+      value: "U.S. NOAA NCEP"
```

Comment:
`globals` need update

IODA convention changed its requirement for `globals`
You can find the valid `global` in [ObsSpae.yaml](https://github.com/JCSDA-internal/ioda/blob/feature/ObsSpace_Validator/share/ioda/yaml/validation/ObsSpace.yaml) in ioda. This is our working branch to add the elements for new variables ([feature/ObsSpace_Validaror](https://github.com/JCSDA-internal/ioda/tree/feature/ObsSpace_Validator))

Here is the example:
https://github.com/NOAA-EMC/spoc/blob/b7428ac1c602314633257fa9f1763e8d389f12d0/dump/mapping/bufr_scatwnd_ascat.yaml#L55-L90